packages/subgraph/tests/superToken/event/superToken.event.test.ts

```diff
@@ -267,6 +267,28 @@ describe("SuperToken Mapper Unit Tests", () => {
             assert.fieldEquals(entityName, id, "liquidationType", liquidationType.toString());
             assert.fieldEquals(entityName, id, "deposit", deposit.toString());
             assert.fieldEquals(entityName, id, "flowRateAtLiquidation", currentFlowRate.toString());
+
+            assertTokenStatisticProperties(
+                null,
+                null,
+                agreementLiquidatedV2Event.address.toHex(),
+                agreementLiquidatedV2Event.block.timestamp,
+                agreementLiquidatedV2Event.block.number,
+                0, // totalNumberOfActiveStreams
+                0, // totalNumberOfClosedStreams
+                0, // totalNumberOfIndexes
+                0, // totalNumberOfActiveIndexes
+                0, // totalSubscriptionsWithUnits
+                0, // totalApprovedSubscriptions
+                BIG_INT_ZERO, // totalDeposit
+                BIG_INT_ZERO, // totalOutflowRate
+                BIG_INT_ZERO, // totalAmountStreamedUntilUpdatedAt
+                BIG_INT_ZERO, // totalAmountTransferredUntilUpdatedAt
+                BIG_INT_ZERO, // totalAmountDistributedUntilUpdatedAt
+                BigInt.fromI32(1000000), // totalSupply = 100
+                3, // totalNumberOfAccounts
+                3 // totalNumberOfHolders
```

Comment:
Use different values if possible, otherwise they might be mixed up and the test wouldn't catch this high likelihood mistake.

Reply:
For the above test case, it is expected to return "3" as totalNumberOfHolders and can't pass anything else or a different value.